src/scene_battle_rpg2k3.cpp

```diff
@@ -1461,8 +1492,14 @@ void Scene_Battle_Rpg2k3::ActionSelectedCallback(Game_Battler* for_battler) {
 	for_battler->SetAtbGauge(0);
 
 	if (for_battler->GetType() == Game_Battler::Type_Ally) {
-		const lcf::rpg::BattleCommand* command = static_cast<Game_Actor*>(for_battler)->GetBattleCommands()[command_window->GetIndex()];
-		for_battler->SetLastBattleAction(command->ID);
+		int index = command_window->GetIndex();
+		// Row command always uses the last index
+		if (index < command_window->GetRowMax() - 1) {
+			const lcf::rpg::BattleCommand* command = static_cast<Game_Actor*>(for_battler)->GetBattleCommands()[index];
+			for_battler->SetLastBattleAction(command->ID);
+		} else {
+			for_battler->SetLastBattleAction(-1);
```

Comment:
This must be verified because the last battle action is a battle page start condition (or was it a condition in a conditional branch?). So this value must match.
Even if the row command can't be picked there through the editor I bet there is at least one game that patched the value and checks for it (see Change Vehicle position with -1)

Reply:
-1 is indeed the Row command, just tested this via LDB-patching :thinking:

Reply:
I'm not sure it's actually -1. The logic I've seen in RPG_RT is usually something like this:

```
auto* batcmd = GetBattleCommandByIndex(window->currentchoice);
if (batcmd) {
  switch(batcmd->type) {
    // Handle each command
  }
} else {
  // Handle row command
}
```

What if you try -2 or a large value greater than the number of configured battle commands?

Reply:
I only tested -1 and -2. -2 didn't activate my event page.

Reply:
Figured it out by now: That's (kinda) a bug. Only kinda because this needs external tools to be utilized:

The ``GetBattleCommandByIndex`` function checks if the cmd idx is ``>= 6`` and if yes it returns ``-1``.
Otherwise it resolves the battle command of the actor. For the row this is ``0``.

Conclusion:
When an actor has 6 battle commands then row has an action ID of -1, otherwise it has an action ID of 0.

@rueter37 can you implement this behaviour? (``SetLastBattleAction(-1)`` for "6 cmds``, else ``SetLastBattleAction(-1)``)

Reply:
Added the LastBattleAction changes.